invenio_rdm_records/services/requests/service.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Community Inclusion Service."""
+
+from invenio_requests import current_requests_service
+from invenio_search.engine import dsl
+
+from invenio_records_resources.services import (
+    Service,
+)
+
+
+class RecordRequestsService(Service):
+    """Service for records' requests.
+
+    The RDM Requests service wraps some operations of the generic requests service,
+    implementing RDM business logic.
+    """
+
+    def search(
+        self,
+        identity,
+        record_pid,
+        params=None,
+        search_preference=None,
+        expand=False,
+        extra_filter=None,
+        **kwargs
+    ):
+        """Search for record's communities."""
```

Comment:
missing record permission check

Reply:
looks good!

For the future: now thinking about this, we might want to override all upper class method, and add a `raise NotImplementedError`...